### PR TITLE
EN-2003-2033 File-Browser: Fix wrong file check and link

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/storage/StorageManagerUtil.java [hunk 1]
```diff
@@ -24,76 +24,101 @@
...
-			_logger.info("Attack avoided - requested path {}", path);
-			return !check;
-		}
-		return check;
-	}
+    private static final String REGEXP_FILE_EXTENSION = "([\\w|\\-]+?$)";
+    private static final String REGEXP_FILE_BASENAME = "\\A(?!(?:COM[0-9]|CON|LPT[0-9]|NUL|PRN|AUX|com[0-9]|con|lpt[0-9]|nul|prn|aux)|[\\s\\.])[^\\\\/:*\"?<>|]{1,254}\\z";
+    private static final String REGEXP_DIR = "(^[\\w|\\.|\\-|\\_|/| ]+?)";
+
+    public static boolean isValidFilename(String fullname) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/storage/StorageManagerUtil.java]: These need tests for master

Comment (eugeniosant) [reply engine/src/main/java/org/entando/entando/aps/system/services/storage/StorageManagerUtil.java]: Done

**Comment by entando-service** [comment]
All is well

**Pull request merged**